pkg/resources/data.go

```diff
@@ -440,7 +440,7 @@ func (d *TemplateData) KubermaticAPIImage() string {
 func (d *TemplateData) parseImage(image string) string {
 	named, _ := reference.ParseNormalizedNamed(image)
 	domain := reference.Domain(named)
-	reminder := reference.Path(named)
+	remainder := reference.Path(named)
```

Comment:
But how will we now remember this?!

Reply:
This will forever *remine* a secret. 

:drum: *ba-dum-tss*